extensions/wikia/DesignSystem/styles/design-system.scss

```diff
@@ -20,6 +20,10 @@ $global-navigation-z-index: 5000101;
 
 .wds-global-navigation-wrapper {
 	height: $wds-global-navigation-height;
+
+	@media screen and (max-width: 768px) {
+		width: 768px;
```

Comment:
I think you could achieve the same with min width, right?

Reply:
yes, but does it make any difference? above 768px there is other breakpoint so this style will not be applied